internal/service/streamprocessor/model_test.go

```diff
@@ -38,6 +39,7 @@ var statsExample = `
 	"memoryTrackerBytes": 0.0,
 	"name": "processor1",
 	"ok": 1.0,
+	"data": { "changeStreamToken": "changeStreamTokenValue" },
```

Comment:
It would be nice to have this as an optional parameter and test both with and without data